instrumentor/controllers/sourceinstrumentation/conditions.go

```diff
@@ -32,7 +33,8 @@ func initiateRuntimeDetailsConditionIfMissing(ic *v1alpha1.InstrumentationConfig
 	}
 
 	// if the workload has no available replicas, we can't detect the runtime
-	if workloadObj.AvailableReplicas() == 0 {
+	if workloadObj.AvailableReplicas() == 0 &&
```

Comment:
what will be the initial message on these sources until we are able to detect the runtime for this job?

should we have some other initial message to indicate what is going on?